Frontend/ui-customer/src/Flow.purs

```diff
@@ -3372,12 +3473,40 @@ myProfileScreenFlow = do
       modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen { data { settingSideBar { opened = SettingSideBarController.CLOSED } } })
       homeScreenFlow
 
+favouriteDriverTripFlow :: FlowBT String Unit
+favouriteDriverTripFlow = do
+  act <- UI.favouriteDriverTrips
+  case act of 
+    GO_TO_FAVOURITE_DRIVER_PROFILE state -> do
+      modifyScreenState $ DriverProfileScreenCommonStateType ( \driverProfileScreen -> driverProfileScreen { props { driverId = fromMaybe "" state.data.driverId } } )
+      driverProfileScreenFlow
+    GO_BACK_TO_SAVED_LOCATION state -> do 
+      void $ lift $ lift $ toggleLoader true
+      resp <- lift $ lift $ Remote.removeFavouriteDriver (fromMaybe "" state.data.driverId)
+      let _ = spy "removeFavouriteDriver" resp
+      void $ lift $ lift $ delay $ Milliseconds 2000.0
+      void $ lift $ lift $ toggleLoader false
+      case resp of
+        Right resp -> do
+          savedLocationFlow HomeScreenFlow
+        Left _ -> do 
+          void $ pure $ toast "Failed To Remove Favourite Ride"
```

Comment:
Added locale string , Please check full PR if english text is hardcoded change it to local string.